src/main/webapp/app/entities/event/index.tsx

```diff
@@ -13,9 +13,10 @@ const Routes = ({ match }) => (
   <>
     <Switch>
       <ErrorBoundaryRoute exact path={`${match.url}/new`} component={EventUpdate} />
+      <ErrorBoundaryRoute exact path={match.url} component={Event} />
+      <ErrorBoundaryRoute exact path={`${match.url}/previous`} component={Event} />
```

Comment:
This line should be deleted as it is not being use anymore.
Yet, can we made the changes in any other ongoing development PR?

Reply:
@junzai97 yes please do it on other PR. Thanks